src/lib.rs

```diff
@@ -61,15 +61,27 @@ impl<'a> From<Model<'a>> for Session {
 }
 
 // ------ ------
-//     Init
+//     Before Mount
```

Comment:
Center the text please. (Same for After Mount).

Reply:
fixed